ipmikvm

```diff
@@ -221,15 +221,26 @@ install_ikvm_application() {
     codebase=$(
       echo "$launch_jnlp" | sed -e '/<jnlp /!d;s/.* codebase="//;s/".*//')
     jar=$(get_jar "$launch_jnlp")
-    linuxlibs=$(
-      echo "$launch_jnlp" | sed -e '
-        /<nativelib.*linux.*x86_64/!d
-        s#.* href="\([^"]*\).jar".*version="\([^"]*\)".*#\1__V\2.jar#
-        s#.* href="\([^"]*\).jar".*#\1.jar#' | sort -u)
+    case "$(uname -s)" in
+        Linux)
+            nativelibs=$(
+              echo "$launch_jnlp" | sed -e '
+                /<nativelib.*linux.*x86_64/!d
+                s#.* href="\([^"]*\).jar".*version="\([^"]*\)".*#\1__V\2.jar#
+                s#.* href="\([^"]*\).jar".*#\1.jar#' | sort -u)
+            ;;
+        Darwin)
+            nativelibs=$(
+              echo "$launch_jnlp" | sed -e '
+                /<nativelib.*libmac.*x86_64/!d
```

Comment:
The only change is this regex right?

Perhaps a:
```
case "$(uname -s)" in
Linux) libprefix=liblinux;;
Darwin) libprefix=libmac;;
*) echo "Unknown platform $(uname -s)" >&2; exit 1;;
esac
```
and
```
nativelibs=$(
  echo "$launch_jnlp" | sed -e '
    /<nativelib.*'$libprefix'.*x86_64/!d
```
and
```
unzip -o $libprefix*.jar
```

Reply:
Yes, simpler. I have updated the code.